modules/ROOT/pages/http/http-authentication.adoc

```diff
@@ -2,357 +2,496 @@
+This example builds and runs an app in Studio that sends a request to the https://developer.github.com/v3[GitHub API] for user information. The GitHub API accepts requests for user information on port 443 to `+https://api.github.com/user+`.
 
-To comply with the basic authentication requirements, the app provides the Github user name and password. You configure the HTTP Request Connector to provide these credentials.
+To comply with the basic authentication requirements, the app provides the GitHub user name and password. You configure the HTTP Request Connector to provide these credentials.
 
-This example requires that you have a Github account.
+This example requires that you have a GitHub account.
 
-*To call the Github API, first set up an HTTP listen connector:*
+To call the GitHub API, first set up an HTTP Listener connector.
 
+////
 [.ex]
 =====
 [discrete.view]
-=== Studio Visual Editor
+////
 
-. In Studio, create a new Mule project: *File* > *New* > *Mule Project*.
-The New Mule Project dialog appears.
-. In Project Settings, set the following options for the HTTP Listen Connector:
+=== Basic Authentication - Studio Visual Editor
+
+. In Studio, create a new Mule project: *File* > *New* > *Mule Project* and set these
+parameters:
++
+** *Project Name*: `myproject`
+** *Runtime*: Select or accept the default `Mule Server 4.1.x` or later.
++
+. In the Mule Palette, click *HTTP*, click the *Listener* operation, and drag 
+the Listener operation to the canvas.
+. In the properties editor for the Listener, set the *Path* to `/` (forward slash).
+. In *Connector Configuration*, click the green plus icon.
 +
-* *Project Name*: `myproject`
-* *Runtime*: Select or accept the default `Mule Server 3.8.0 EE` or later.
-. Drag an HTTP component from the palette to the Source section of the flow.
-. In the properties editor, accept the default *Path* `/` and set *Allowed Methods* to GET.
-. In Connector Configuration, click image:http/add-16x16.png[Add-16x16].
+The *HTTP Listener config* dialog appears.
 +
-The HTTP Listen Configuration dialog appears.
 . Click OK to accept the following options:
 +
-* Name: HTTP_Listen_Configuration
-* Protocol: HTTP
-* Host: 0.0.0.0
-* Port: 8081
+** *Name*: `HTTP_Listener_config`
+** *Protocol*: `HTTP`
+** *Host*: `0.0.0.0`
+** *Port*: `8081`
 +
 . Click OK.
+. In the properties editor, click the *Advanced* tab and set *Allowed methods* to `GET`.
 . Save changes.
 +
 The error indicator disappears.
 
-*Next, set up an HTTP Request connector:*
+=== Basic Authentication - Configure an HTTP Request Connector
 
-. Drag another HTTP connector from the palette, and drop it in the Process area of the flow.
-. In the properties editor, in Connector Configuration, click image:http/add-16x16.png[Add-16x16].
+. Drag an *HTTP* > *Request* operation from the Mule Palette to the Process area of the Studio flow. This operation is known as the HTTP Request connector.
```

Comment:
Remove `This operation is known as the HTTP Request connector`. Instead, add a link to the http request operation documentation page